### DISCUSSION TITLE
Does it work on SDXL or Forge?

### DISCUSSION THREAD

**Discussion by LucianoDaluz** [glucauze/sd-webui-faceswaplab#172] [General]
Just installed on Forge, and the Faceswaplab tab doesnt show up. Does anybody knows if it doesnt work on Forge or if we need to do something in order to make it work? Thanks

**Comment by LucianoDaluz** [comment]
Update, yeah it works

**Comment by coffeedrinker0725** [comment]
faceswaplab 1.2.7 is ok when running on stable difussion webui (A1111). However, running on stable difussion webui forge, it is different: in case of txt2img, it ok. but in case of img2imge, it struck attribute error: FaceSwapLab - ERROR - Failed to swap face in postprocess method : 'tuple' object has no attribute 'save'. faceswaplab 1.2.7 is already the latest version.
